### PR TITLE
Add CSS Units guide

### PR DESCRIPTION
**Pull request by agustito37**
CSS Units are confusing and misusing them have a direct impact in the app quality; so I added a small table to have a reference.

### CODE REVIEW

**Comment by agustito37** [comment]
Yes, you are right.

File: css.md [hunk 1]
```diff
@@ -2,3 +2,19 @@
...
 We follow https://cssguidelin.es. In the css naming convention, we are aligned with the Hyphen Delimited convention instead of BEM-like one.
 For SASS, we follow https://sass-guidelin.es/
+
+## CSS Units
+
+Refer to this table to decide which unit to use in each case.
+
+| Unit       | Use for                                                          |
+|:-----------|:-----------------------------------------------------------------|
+| rem        | font-size, dimensions (margin, padding, height)                  |
```
Comment (mdamonte88) [inline css.md]: I agree that font-size could be only in rem but other properties as (margin padding, etc) could be in em, px.
Furthermore is good to know that rem is using based on the font-size to the root element.

File: css.md [hunk 2]
```diff
@@ -2,3 +2,19 @@
...
 We follow https://cssguidelin.es. In the css naming convention, we are aligned with the Hyphen Delimited convention instead of BEM-like one.
 For SASS, we follow https://sass-guidelin.es/
+
+## CSS Units
+
+Refer to this table to decide which unit to use in each case.
+
+| Unit       | Use for                                                          |
+|:-----------|:-----------------------------------------------------------------|
+| rem        | font-size, dimensions (margin, padding, height, width)           |
```
Comment (mdamonte88) [inline css.md]: I agree that font-size could be only in rem but other properties as (margin padding, etc) could be in em, px.
Furthermore is good to know that rem is using based on the font-size to the root element.

Comment (agustito37) [reply css.md]: Sure, I agree; this is just a reference, if you have a valid reason to use another unit, it's fine too.

**Pull request merged**